sdk/storage/storage-blob/src/BlobLeaseClient.ts

```diff
@@ -101,7 +98,7 @@ export interface LeaseOperationOptions extends CommonOptions {
  * @export
  * @class LeaseClient
```

Comment:
@class LeaseClient => @class BlobLeaseClient

Reply:
Fixed with https://github.com/Azure/azure-sdk-for-js/pull/5719/commits/152cb2a32514682da8ed5ce1fb3ad42c0f87983d has the c